impeller/geometry/path.cc

```diff
@@ -73,6 +73,17 @@ void Path::Shift(Point shift) {
   }
 }
 
+Path Path::Clone() const {
+  Path new_path;
+  new_path.components_ = components_;
+  new_path.contours_ = contours_;
+  new_path.computed_bounds_ = computed_bounds_;
+  new_path.convexity_ = convexity_;
+  new_path.points_ = new_path.points_;
+  new_path.fill_ = fill_;
+  return new_path;
+}
```

Comment:
Can we make the copy constructor private and call that here?  Then we don't have to worry about this atrophying.

Reply:
Ahh TIL!